### DISCUSSION TITLE
Slider

### DISCUSSION THREAD

**Discussion by Febakke** [digdir/designsystemet#1823] [Components]
### Describe the component

For bruk der man vet på forhånd område, men ønsker at bruker selv skal begrense utvalget. 

Se f.eks eksempler fra designsystem på finn.no linket nedenfor

### If you have any examples or similar components feel free to include links or screenshots here

https://carbondesignsystem.com/components/slider/usage/
https://m3.material.io/components/sliders/overview
https://www.fabric-ds.io/slider

Issue: #1197

**Comment by oddvernes** [comment]
Chrome har begynt å [prototype](https://groups.google.com/a/chromium.org/g/blink-dev/c/QbFxuQM1UHU/m/UBHtDtFgCQAJ) nytt `<rangegroup>` element som vil kollapse en liste med input type=range til en range med flere thumbs 

[open-ui explainer](https://open-ui.org/components/enhanced-range-input.explainer/#multi-handle-support-with-rangegroup)

[webkomponent poc](https://brechtdr.github.io/enhanced-range-slider-poc/)
